src/working-here/supporting-your-development-and-wellbeing.md

```diff
@@ -184,14 +182,13 @@ week basis to ensure you are receiving everything you need, and will NEVER get
 bored of your questions!
 
 If you feel that you’re not getting as much time with your helper as you’d like,
-please speak to the People Manager.
+please speak to the People team.
 
 As a helper, you will be vital in ensuring you have the regular catch ups - if
 you are finding it hard to schedule them, please do talk to Bus Ops.
 
 For more information on being a helper, please see the
-[checklist](https://docs.google.com/document/d/1QPv7Mg7TTlt_OyLRtstAtPQBcId-ct-uTbWqYnso2jI)
-or speak to the People Manager.
+[checklist ](https://docs.google.com/document/d/1QPv7Mg7TTlt_OyLRtstAtPQBcId-ct-uTbWqYnso2jI)or speak to the People team.
```

Comment:
Looks like the spacing has gone awry here for some reason. It should be `[checklist](https://docs.google.com/document/d/1QPv7Mg7TTlt_OyLRtstAtPQBcId-ct-uTbWqYnso2jI) or speak to the People team.` if it's on one line. The link still works, but it's also linking the space between the end of `checklist` and the start of `or`